onthequad/onthequad/UserEndPoints.py

```diff
@@ -0,0 +1,87 @@
+import falcon
+import json
+from .UsersDB import UsersDB
+from .DataBaseSetUp import DataBaseSetUp
+authen = DataBaseSetUp.authentication()
+
+class UserEndPoints:
+
+    def on_get(self, req, resp):
+        
+        send = UsersDB.getAllUsers()
+        resp.body = json.dumps(send)
+        resp.status = falcon.HTTP_200
+
+    def on_post(self, req, resp):
+        data = json.loads(req.stream.read())
+    
+        if any(data):
+
+            firstName = data['firstName']
```

Comment:
I know how to split the email to get the username, but I want to use create an account by email and password because firebase has that built in, it will handle all the validation, password encryption etc.  I can even add a feature that sends an email to verify the user later. That why I did it this way since user by the username from the email and email address is almost the same thing. But if we want to do username, we need to handle all encryption etc. I will discuss more with the team about this. I already sent a message on slack about this. You may miss it.

Reply:
I dont see any usernames in the database, thats why assumed we didnt have that as a functionality.